src/html-duration-picker.js

```diff
@@ -208,12 +188,7 @@ export default (function() {
       const totalPickerWidth = currentPickerStyle.width;
       picker.setAttribute('data-upgraded', true);
       picker.value = '00:00:00';
-      picker.style.textAlign = 'right';
-      picker.style.paddingRight = '20px';
-      picker.style.boxSizing = 'border-box';
-      picker.style.width = '100%';
-      picker.style.margin = 0;
-      picker.style.cursor = 'text';
+      picker.setAttribute('style', 'text-align: right; padding-right: 20px; box-sizing: border-box; width: 100%; margin: 0; cursor: text;');
```

Comment:
This section, we want to avoid using `setAttribute('style'...` as this will override the existing styles. So if another developer had applied different styles to the picker, we would like to retain those styles, and then addon our styles only as necessary

Reply:
I reverted that section.